### DISCUSSION TITLE
Unclear license situation

### DISCUSSION THREAD

**Discussion by cz-dev-ge** [atifaziz/Fizzler#83] [Q&A] [answered]
Hi there,
I noticed that COPYING.txt and the license file in the nuget package contain both GPL and LGPL. Is it and OR or AND relation between them for this project? Or: can I chose which one to use or do both apply to different parts of the software?

Best regards.

**Comment by atifaziz** [comment, marked as answer]
It is LGPL. The GPL is included because the LGPL is a set of additional permissions on top of the former; thus makes references to the GPL. You need both to understand the LGPL.

**Reply by cz-dev-ge** [reply]
Thanks for your explanation.

**Reply by cz-dev-ge** [reply]
You could consider putting only the LGPL into LICENSE.txt and putting GPL into a LICENSE_GPLv3.txt. That way GitHub automatically tags this repo as LGPL. Best regards.

**Comment by bilbothebaggins** [comment]
I got here 2024 because I was similarly confused (or rather, our license scanner got confused) by the situation.
(I see that the FSF itself lists such a combined document on https://www.gnu.org/licenses/lgpl+gpl-3.0.txt - not helpful IMHO.)

A README clarification or some such would really help.

The _nuget package_ only contains the license file and no source code - it is even unclearer in that context!

For Nuget, you could consider switching to a [license expression](https://learn.microsoft.com/en-us/nuget/reference/nuspec#license) instead of a license file ref - in that case, it would at least be clear from the nuget side.

Cheers.

**Reply by jwfx** [reply]
Pretty much ran into the same problem.

I would also welcome the use of a license expression for the NuGet package. 

@atifaziz 
Just in case you are not already aware: A number of LGPL related expressions have been deprecated (see https://spdx.org/licenses/ bottom of the page). So picking one that is still supported would probably make sense.

**Reply by atifaziz** [reply]
@bilbothebaggins, @jwfx I hear you. Tracking this in issue #86.

**Reply by atifaziz** [reply]
This should now be addressed with [release 1.3.1](https://github.com/atifaziz/Fizzler/releases/tag/v1.3.1). The [nuget.org page for the release](https://www.nuget.org/packages/Fizzler/1.3.1) now clearly indicates the license:

<img width="163" alt="image" src="https://github.com/atifaziz/Fizzler/assets/20511/2fccf965-aba7-4776-a1a8-738e4ef04ce9">

**Reply by jwfx** [reply]
Looks good, thanks for the quick reaction. 👍
